src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -265,6 +265,19 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		};
 		this._requestedCompletionsIndex = this._currentPromptInputState.cursorIndex;
 
+		// Show loading indicator before making async completion request
+		const suggestWidget = this._ensureSuggestWidget(terminal);
+		const dimensions = this._getTerminalDimensions();
+		if (dimensions.width && dimensions.height) {
+			const xtermBox = this._screen!.getBoundingClientRect();
+			const cursorPosition = {
+				left: xtermBox.left + terminal.buffer.active.cursorX * dimensions.width,
+				top: xtermBox.top + terminal.buffer.active.cursorY * dimensions.height,
+				height: dimensions.height
+			};
+			suggestWidget.showTriggered(!explicitlyInvoked, explicitlyInvoked ? 50 : 250, cursorPosition);
```

Comment:
This passes in 250 for not explicitly but I don't think we want to show it at all on that case as it would get annoying.